README.rst

```diff
@@ -5,5 +5,5 @@ We will try to keep this up-to-date with pythonnet and upstream changes that mig
 Changes relative to pythonnet:
 
 * Revert of `#1240 <https://github.com/pythonnet/pythonnet/pull/1240>`_.
-* Enum REPR `#2239 <https://github.com/pythonnet/pythonnet/pull/2239>`_ is included in this release of version 3.0.2, but is unreleased in pythonnet
```

Comment:
the enum repr note is no longer valid because a new version of pythonnet was released